### ISSUE TITLE
Google Cast function not working using the keyboard

### ISSUE THREAD

**Issue by mdaoust-sidlee** [jwplayer/jwplayer#4005]
### Expected Behavior

*To improve accessibility, user should be able to use the keyboard to activate the casting function.*

### Actual Behavior

*Currently using the keyboard, it is impossible to activate the casting function, neither with the "enter" key nor with the "space" key. At the moment, casting only works on click.*

### Steps to reproduce

1. Having a Google Chromecast available on your network.
2. Use the tab key to navigate and focus on the "Cast" button.
3. Press "enter" or "space" should trigger the Casting function.

### Notes

*Unfortunately, I don't have access to Airplay to confirm or not if the problem is also present with Apple product.*

### Environment
- *Windows 11*
- *Chrome 108*
- *JW Player 8.26.4*
